addons/addon-base/packages/services/lib/helpers/system-context.js

```diff
@@ -31,7 +29,7 @@ function getSystemRequestContext() {
 
   const systemUid = '_system_';
   const systemUsername = '_system_';
-  const systemUserNamespace = internalAuthProviderId;
+  const systemUserNamespace = 'cognito';
```

Comment:
I think we can choose our own preset value for this. An example `ns` value for an Auth0 user is `Auth0||||https://cognito-idp.eu-west-3.amazonaws.com/eu-west-3_U204NRMAr",`.

To keep things simple for a system user, I'm thinking we can just use `cognito`. Let me know if you prefer another default name for system users.

Reply:
Just a thought - If this value doesn't need to be anything specific out of the actual authentication providers, then maybe we can just set it to `system` or `lambda` (since we won't be able to find this user in cognito anyways).

Reply:
Noted: renaming to `_system_` to be consistent with `systemUid` and `systemUserNamespace`